kickstart/finish.erb

```diff
@@ -34,7 +34,7 @@ echo 'root:<%= root_pass -%>' | /usr/sbin/chpasswd -e
 <% end -%>
 
 <% if puppet_enabled -%>
-<% if @host.param_true?('enable-puppetlabs-repo') -%>
+<% if @host.params['enable-puppetlabs-repo'] -%>
```

Comment:
the problem here is that global(!) paramters in Foreman are strings, not booleans, so also their value has to be checked,  try for yourself :balloon::
```ruby
foo = 'true'
bar = 'false'

if foo
  puts 'foo is true'
end

if bar
  puts 'bar is true'
end
```